static/less/simulator.less

```diff
@@ -272,3 +272,46 @@
     display: none;
   }
 }
+
+.button-reply{
```

Comment:
Can we see some screenshots of the simulator changes

Reply:
Yeah, sure.

![screen shot 2017-08-25 at 13 34 28](https://user-images.githubusercontent.com/5198870/29723248-371b8bb0-899a-11e7-9c24-5b9b929c47e8.png)